### DISCUSSION TITLE
Docusaurus plugin

### DISCUSSION THREAD

**Discussion by ArtFlag** [scalar/scalar#1011] [Ideas]
[Docusaurus](https://github.com/facebook/Docusaurus) is picking up traction as an open-source docs renderer. It's used by Figma, React, and many other famous companies. 

Docusaurus has a plugin feature, and someone already created a [plugin for Redocly](https://github.com/rohit-gohri/redocusaurus), which is great. 

I think it might be a good idea to consider building a plugin for Scalar, to boost the adoption of your tool inside Docusaurus 👍

**Comment by marclave** [comment]
love this idea @ArtFlag ! we have chatted about it internally and it's defs on our roadmap ✨

**Comment by marclave** [comment]
@ArtFlag we shipped the plugin today ✨ 

excited for y'all to try it out and give it a spin ✨ 

https://github.com/scalar/scalar?tab=readme-ov-file#with-docusaurus

**Comment by ArtFlag** [comment]
Wow, that was quick, thanks @marclave! 

First feedback: I can't seem to make it work using the provided sample 😅 I get a `page not found` on `/scalar`. Not sure how to proceed to fix this. Any clue? (using Docusaurus 3.1.1)

Also does path take local files too?

**Reply by marclave** [reply]
@ArtFlag currently we do not load local files, but we can add that!

Are you able to share a repository or small snippet so I can make sure it works for your use case :)

in the meantime you should be able to serve the spec file based on a URL if its served in a public directory in docusaurus

**Reply by ArtFlag** [reply]
Actually, I take this back, the problem I was facing was between the chair and the keyboard 🙃  

Thanks a lot for creating the plugin so quickly!! 🙌
